src/test_scenes/sceneTransformations.js

```diff
@@ -18,7 +18,7 @@ class SceneTransformations extends React.Component {
 
         <view localPosition={[0, 0.1, 0]}>
```

Comment:
Change to import { View } from "magic-script-components";

Reply:
Same here